lib/keyboard_utils.dart

```diff
@@ -71,22 +71,22 @@ class KeyboardUtils {
     _listenersKeyboardEvents.clear();
   }
 
-  void _onKeyboardListener(Object data) {
+  void _onKeyboardListener(Object? data) {
     if (data != null && data is String) {
       try {
         final Map<String, dynamic> keyboardOptionsMap = jsonDecode(data);
         _keyboardOptions = KeyboardOptions.fromJson(keyboardOptionsMap);
 
         _listenersKeyboardEvents.forEach((_, final KeyboardListener listener) {
-          if (_keyboardOptions.isKeyboardOpen &&
+          if (_keyboardOptions!.isKeyboardOpen! &&
               listener.willShowKeyboard != null) {
-            listener.willShowKeyboard(_keyboardOptions.keyboardHeight);
+            listener.willShowKeyboard!(_keyboardOptions!.keyboardHeight);
             return;
           }
 
-          if (!_keyboardOptions.isKeyboardOpen &&
+          if (!_keyboardOptions!.isKeyboardOpen! &&
               listener.willHideKeyboard != null) {
-            listener.willHideKeyboard();
+            listener.willHideKeyboard!();
           }
         });
       } on dynamic catch (_) {}
```

Comment:
Because of null-safety dynamic can throw a null type and 'on' clause don't accept it because it isn't valid to throw a nullable expression.
```suggestion
      }  catch (_) {}
```